.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
           apps: sbt
 
       - name: Run benchmarks
-        run: sbt "++2.13.10 benchmarks/Jmh/run -rf json -rff $(pwd)/$(git describe --tags --always).json" # TODO LB scala version from matrix
+        run: sbt "++2.13.11 benchmarks/Jmh/run -rf json -rff $(pwd)/$(git describe --tags --always).json" # TODO LB scala version from matrix
```

Comment:
scala version seems to be redundant, as `benchmarks` is now managed by sbt-projectmatrix

Reply:
fixed